theano/gpuarray/linalg.py

```diff
@@ -480,9 +471,6 @@ def make_node(self, x):
             raise LinAlgError("Matrix rank error")
         return theano.Apply(self, [x], [x.type()])
 
```

Comment:
(Github does not allow me to comment in the right line)

As you require x to be C contiguous in C code (magma_inv.c), you should add `x = gpu_contiguous(x)` after `x = as_gpuarray_variable(x, ctx_name)`.